packages/headless/src/features/insight-interface/insight-interface-slice.test.ts

```diff
@@ -0,0 +1,161 @@
+import {fetchInterface} from './insight-interface-actions';
+import {insightInterfaceReducer} from './insight-interface-slice';
+import {
+  getInsightInterfaceInitialState,
+  InsightInterfaceState,
+} from './insight-interface-state';
+
+describe('insight interface slice', () => {
+  const requestId = 'some-request-id';
+  const errorResponse = {
+    message: 'something bad happened',
+    statusCode: 400,
+    type: 'badluck',
+  };
+
+  let state: InsightInterfaceState;
+
+  beforeEach(() => {
+    state = getInsightInterfaceInitialState();
```

Comment:
if I understood it correctly, you're never reassigning state, right? I'd suggest getting rid of this and calling `getInsightInterfaceInitialState()` whenever you need the initial state.  Otherwise it makes it a bit confusing to understand that `state` is always the initial state

Reply:
Thanks for pointing that out! You are totally right, `state` is never reassigned. I'll rework this part to make it clearer.